wechatpy/work/services/__init__.py

```diff
@@ -0,0 +1,112 @@
+# -*- coding: utf-8 -*-
+import json
+import time
+import logging
+import requests
+from wechatpy.client.base import BaseWeChatClient
```

Comment:
```suggestion

from wechatpy.client.base import BaseWeChatClient
```

标准库、第三方库和自己的 import 可以用空行分割下。